src/VirtualHostGenerator.php

```diff
@@ -127,7 +127,10 @@ private function buildServerHost(string $hostname, string $admin, array &$virtua
     }
     public function create()
     {
-        exec("service apache2 stop");
+        exec("a2ensite https-only");
```

Comment:
why is https-only enabled and disabled in the end? http-only gets disabled and enabled

Reply:
fixed, forgot to change this case